src/functions/module_math.ts

```diff
@@ -1317,3 +1318,68 @@ export const TRUNC: AddFunctionDescription = {
   },
   isExported: true,
 };
+
+export const MUNIT: AddFunctionDescription = {
+  description: _lt("Return an n*n identity matrix."),
+  args: [arg("n (number)", _lt("size of the unit matrix"))],
+  returns: ["RANGE<NUMBER>"],
+  compute: function (n: PrimitiveArgValue): number[][] {
+    return Array.from({ length: toNumber(n) }, (_, i) =>
+      Array.from({ length: toNumber(n) }, (_, j) => (i == j ? 1 : 0))
+    );
+  },
+  isExported: true,
+};
+
+export const MFILL: AddFunctionDescription = {
```

Comment:
Don't forget to remove the function, or at leat don't export it before merging. I'm pretty sure this doesn't exist in Excel.